src/lib.rs

```diff
@@ -115,9 +117,13 @@ pub(crate) const REQWEST_TIMEOUT: Duration = Duration::from_secs(10);
 
 /// Placing the main function in lib.rs allows other crates to import it and embed Lemmy
 pub async fn start_lemmy_server(args: CmdArgs) -> Result<(), LemmyError> {
-  let scheduled_tasks_enabled = !args.disable_scheduled_tasks;
   let settings = SETTINGS.to_owned();
 
+  let mut startup_server_handle = None;
```

Comment:
Might want to leave a comment either here, or on the function, that this will return a default error, while waiting for long-running sync jobs like `run_migrations` and `run_advanced_migrations` . 

Otherwise its not immediately clear what this is for.

Reply:
Makes sense, but phiresky merged too quickly :D

https://github.com/LemmyNet/lemmy/pull/4022